docs/modules/development/pages/rest/rest-api.adoc

```diff
@@ -1,121 +1,201 @@
+
 [[rest-api]]
 = REST API
 
-A RESTful interface is a web service conforming to the REST architectural style as described in link:http://oreilly.com/catalog/9780596529260[_RESTful Web Services_].
+A RESTful interface is a web service conforming to the REST architectural style as described in http://oreilly.com/catalog/9780596529260[_RESTful Web Services_].
 This section describes the RESTful interface for {page-component-title}.
 
-== ReST URL
-
-The base URL for REST calls is: http://opennmsserver:8980/opennms/rest/
+== REST URL
 
-For instance, http://localhost:8980/opennms/rest/alarms/ will give you the current alarms in the system.
+The base URL for REST calls is `http://opennmsserver:8980/opennms/rest/`.
+For instance, `http://localhost:8980/opennms/rest/alarms/` will give you the current alarms in the system.
 
 == Authentication
 
 Use HTTP basic authentication to provide a valid username and password.
-By default you will not receive a challenge, so you must configure your REST client library to send basic authentication proactively.
+By default, you will not receive a challenge, so you must configure your REST client library to send basic authentication proactively.
 
-== Data Format
+== Data format
 
-Jersey lets REST calls to be made using either XML or JSON.
-By default, returns a request to the API in XML.
-XML is delivered without namespaces.
+Jersey enables {page-component-title} to create REST calls using either XML or JSON.
+By default, it returns a request to the API as XML data, delivered without namespaces.
 
-NOTE: If a namespace is added manually, in order to use a XML tool to validate against the XSD (like xmllint) it won't be preserved when OpenNMS updates that file.
-      The same applies to comments.
+NOTE: If a namespace is added manually in order to use a XML tool to validate against the XSD (for example, xmllint), it won't be preserved when OpenNMS updates that file.
```

Comment:
This sentence is difficult to parse. To clarify: are there other scenarios when a namespace might be added manually and if so, will it still not be updated. OR does this behaviour happen when the namespace is added manually in this particular situation?

Reply:
From what I understand, a user may add a namespace manually in order to use an XML tool for validations. In cases where the namespaces are manually added, they won't be preserved when OpenNMS updates the file containing it.

Reply:
I suggest rephrasing with what you just wrote to make it clearer:

```suggestion
NOTE: If a namespace is added manually (for example, to use an XML tool like xmllint to validate against the XSD), it won't be preserved when OpenNMS updates the file that contains that namespace.
```